scripts/fetch-dnf-cache.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+#
+# Fetch dnf cache for the default set of container images
```

Comment:
Will we have a similar script for testcloud images? It might be good to make clear this script takes care of podman images only, `fetch-dnf-cache.sh` is quite generic.

Reply:
Understood, yeah, there might be more in the future. So... what about `prepare/podman-images.sh` instead? Updated in 3fe95b6e.